lib/checkPullRequestTemplate.js

```diff
@@ -203,34 +252,40 @@ const checkTemplate = async (context) => {
 
   const explanationMessage = checkPRExplanation(pullRequest);
   const checklistMessage = checkEssentialChecklist(pullRequest);
+  const proofOfChangesMessage = checkProofThatChangesAreCorrect(pullRequest);
   let commentBody = '';
-  if (explanationMessage && checklistMessage) {
-    commentBody =
-      'Hi @' +
-      pullRequest.user.login +
-      ', ' +
-      explanationMessage +
-      '<br>Also, ' +
-      checklistMessage.trim() +
-      ' Thanks!';
-  } else if (explanationMessage) {
-    commentBody =
-      'Hi @' +
-      pullRequest.user.login +
-      ', ' +
-      explanationMessage +
-      ' Thanks!';
-  } else if (checklistMessage) {
-    commentBody =
+
+  count = 1;
+
+  commentBodybasic = commentBody =
       'Hi @' +
       pullRequest.user.login +
-      ', ' +
-      checklistMessage.trim() +
-      ' Thanks!';
-  } else {
+      ', can you complete the following:\n';
+
+  if (explanationMessage) {
+    commentBody += count.toString() +
```

Comment:
`count.toString()` is not required when concatenating strings.

Reply:
DONE